tests/test_progressbar.py

```diff
@@ -1,7 +1,13 @@
 # Copyright (c) Open-MMLab. All rights reserved.
 import sys
+import os
```

Comment:
`import os` should be the first statement according to alphabet.

Reply:
why does pre-commit not doing this automatically?